miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultRunService.java

```diff
@@ -355,6 +356,10 @@ private void validateChanges(Run before, Run changed) throws IOException {
       errors.add(new ValidationError("sequencingKitId", "Must be a sequencing kit"));
     }
 
+    if (((before == null && changed.isDataApproved() != null) || isChanged(Run::isDataApproved, changed, before))
+        && authorizationManager.getCurrentUser().getGroups().stream().noneMatch(group -> Group.RUN_APPROVERS.equals(group.getName()))) {
```

Comment:
or admin? I guess this is a policy question and not a technical one...

Reply:
:+1: Changed so admin can approve runs too